server/src/main/resources/db/mysql/migration/V1__create_tables.sql

```diff
@@ -40,8 +40,8 @@ CREATE TABLE clients (
 
 CREATE TABLE accessgrants (
   id int NOT NULL AUTO_INCREMENT,
-  groupid int REFERENCES groups(id) ON DELETE CASCADE,
```

Comment:
Flyway uses checksums on the file contents and will get confused. We should avoid changing existing `.sql` files unless it's something completely broken. In this case, I would just add a new migration to remove the foreign key and call it a day.